Doxyfile.in

```diff
@@ -1158,7 +1158,7 @@ INCLUDE_FILE_PATTERNS  =
 # undefined via #undef or recursively expanded use the := operator 
 # instead of the = operator.
 
-PREDEFINED             = 
+PREDEFINED             = DOXYGEN
```

Comment:
I assume just

    PREDEFINED = ENABLE_DEBUG_DRAWINGS

would work as well?

Reply:
Yes, but that would not match the setup used in the repository. The ENABLE_DEBUG_DRAWINGS #define is not set globally. I think it is better to PREDEFINE the well known DOXYGEN preprocessor variable instead.